ipie/utils/chunk_large_chol.py

```diff
@@ -7,7 +7,7 @@ def split_cholesky(ham_filename, nmembers):
     #
     with h5py.File(ham_filename, "r") as source_file:
         dataset = source_file["LXmn"][()]  # pylint: disable=no-member
```

Comment:
Why (), I thought () was for accessing scalar data sets?

Reply:
I think the difference between [()] and [:] is mostly stylistic, I usually used [()] to retrieve all slices from h5 file to a numpy array.